files/zh-cn/web/http/status/403/index.html

```diff
@@ -12,7 +12,7 @@
 
 <p>状态码 <strong><code>403 Forbidden</code></strong> 代表客户端错误，指的是服务器端有能力处理该请求，但是拒绝授权访问。</p>
 
-<p>这个状态类似于 {{HTTPStatus("401")}}，但进入该状态后不能再继续进行验证。该访问是长期禁止的，并且与应用逻辑密切相关（例如不正确的密码）。</p>
+<p>这个状态类似于 {{HTTPStatus("401")}}，但进入 {{HTTPStatus("403")}}状态后不能再继续进行验证。该访问是长期禁止的，并且与应用逻辑密切相关（例如没有足够的权限访问该资源）。</p>
```

Comment:
@gray19950103 
[原文](https://github.com/mdn/content/blob/dc9f7fd11b7ddff380d9293fa53bc7a536cbc7f7/files/en-us/web/http/status/403/index.md?plain=1#L15) `re-authenticating makes no difference` 的意思是重新驗證也不會有差異。參考以下翻譯？
```
但 403 状态码意谓重新进行验证也不会改变现况。
```

Reply:
@irvin 
嗯，这句话确实和原文有差异，之前我没有注意到。看看下面这样ok吗？
```suggestion
<p>这个状态类似于 {{HTTPStatus("401")}}，但进入 {{HTTPStatus("403")}}状态后即使重新验证也不会改变该状态。该访问是长期禁止的，并且与应用逻辑密切相关（例如没有足够的权限访问该资源）。</p>
```